diff --git a/node_modules/gatsby-transformer-react-docgen/parse.js b/node_modules/gatsby-transformer-react-docgen/parse.js
index b8a5702..985bbcd 100644
--- a/node_modules/gatsby-transformer-react-docgen/parse.js
+++ b/node_modules/gatsby-transformer-react-docgen/parse.js
@@ -9,9 +9,9 @@ var _objectWithoutPropertiesLoose2 = _interopRequireDefault(require("@babel/runt
 
 var _codeFrame = require("@babel/code-frame");
 
-var _reactDocgen = require("react-docgen");
+var _reactDocgen = require("tt-react-docgen");
 
-var _parse = require("react-docgen/dist/parse");
+var _parse = require("tt-react-docgen/dist/parse");
 
 var _doclets = require("./doclets");
 
